index.js

```diff
@@ -0,0 +1,10 @@
+import Book from './modules/book.js';
+import navigation from './modules/nav.js';
+import * as dateTime from './modules/date.js';
```

Comment:
- [OPTIONAL] According to the project requirements, the module should be imported into your js file. eg: `import { DateTime } from "luxon";`